base_layer/core/src/transactions/types.rs

```diff
@@ -21,97 +21,10 @@
 // USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 use crate::transactions::bullet_rangeproofs::BulletRangeProof;
-use std::sync::Arc;
-use tari_crypto::{
-    common::Blake256,
-    ristretto::{
-        dalek_range_proof::DalekRangeProofService,
-        pedersen::{PedersenCommitment, PedersenCommitmentFactory},
-        RistrettoComSig,
-        RistrettoPublicKey,
-        RistrettoSchnorr,
-        RistrettoSecretKey,
-    },
-};
-
-/// Define the explicit Signature implementation for the Tari base layer. A different signature scheme can be
-/// employed by redefining this type.
-pub type Signature = RistrettoSchnorr;
-/// Define the explicit Commitment Signature implementation for the Tari base layer.
-pub type ComSignature = RistrettoComSig;
-
-/// Define the explicit Commitment implementation for the Tari base layer.
-pub type Commitment = PedersenCommitment;
-pub type CommitmentFactory = PedersenCommitmentFactory;
-
-/// Define the explicit Secret key implementation for the Tari base layer.
-pub type PrivateKey = RistrettoSecretKey;
-pub type BlindingFactor = RistrettoSecretKey;
-
-/// Define the hash function that will be used to produce a signature challenge
-pub type SignatureHasher = Blake256;
-
-/// Define the explicit Public key implementation for the Tari base layer
-pub type PublicKey = RistrettoPublicKey;
-
-/// Specify the Hash function for general hashing
-pub type HashDigest = Blake256;
-
-/// Specify the digest type for signature challenges
-pub type Challenge = Blake256;
-
-/// The type of output that `Challenge` produces
-pub type MessageHash = Vec<u8>;
+use tari_crypto::ristretto::dalek_range_proof::DalekRangeProofService;
 
 /// Specify the range proof type
 pub type RangeProofService = DalekRangeProofService;
 
 /// Specify the range proof
 pub type RangeProof = BulletRangeProof;
```

Comment:
Is there a reason why you left these to here and not move them over to comment::types as well?
`MAX_RANGE_PROOF_RANGE` is moved to common, so I would expect that to live with the other rangeproof types.

Reply:
I originally thought it had much deeper integration in core, but I've managed to merge bullet_proof into common_types as well